test/jdk/sun/java2d/loops/CopyAreaSpeed.java

```diff
@@ -111,158 +129,15 @@ public void run()
             }
             try {
                 Thread.sleep(1);
```

Comment:
Maybe because of Graphics paint and copyArea ?

Reply:
It is still a little bit suspicious because adding fixed delay into the test loop makes the test less accurate the fastest hardware gets but i guess in order to maintain the test value compared to the previous versions of the test it will be wrong to change time calculations.